server/repository/src/migrations/v1_01_11/name_indexes.rs

```diff
@@ -0,0 +1,15 @@
+use crate::{migrations::sql, StorageConnection};
+
+pub(crate) fn migrate(connection: &StorageConnection) -> anyhow::Result<()> {
+    sql!(
+        connection,
+        r#"
+        CREATE INDEX IF NOT EXISTS "index_name_first_name" ON "name"("first_name"); 
+        CREATE INDEX IF NOT EXISTS "index_name_last_name" ON "name"("last_name"); 
+        CREATE INDEX IF NOT EXISTS "index_name_code" ON "name"("code"); 
+        CREATE INDEX IF NOT EXISTS "index_name_national_health_number" ON "name"("national_health_number"); 
```

Comment:
I don't have a national_health_number in develop. is this just on feature branch?

Reply:
good spot! thanks! I will write an issue to diff against develop and move additional fields into separate migrations

Reply:
#1578